rtl_wmbus.c

```diff
@@ -848,6 +855,9 @@ static void process_options(int argc, char *argv[])
         case 'v':
             opts_show_used_algorithm = 1;
             break;
+        case 'V':
+            print_version();
```

Comment:
exit(EXIT_FAILURE); is missing here.

Reply:
But, it reports exit(EXIT_SUCCESS) below? Do you mean that it is a standard to report failure when printing the version number successfully?

Reply:
I mean it should exit with success, now if you run it with `-V` option it gets stuck.

Reply:
Ah, but are you looking at the earlier version of the pull-request?
The latest version does have the exit. I can repush with force if you like.

Reply:
I see it now. No need to force-push.